profile/CSIP.xml

```diff
@@ -1182,9 +1182,8 @@
             </requirement>
             <requirement ID="CSIP101" REQLEVEL="SHOULD" EXAMPLES="structMapExample1">
                 <description>
-                    <head>File division</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The transferred files referenced in the file section file group are described in the structural map in a single sub division.</p>
-                    <p xmlns="http://www.w3.org/1999/xhtml">When the transfer comprises of only data and no representations there should still be a single empty representation div present.</p>
+                    <head>Content division</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">When there are no representations present the content referenced in the file section file group named "Representations" is described in the structural map in a single sub division.</p>
```

Comment:
When no representations are present the content referenced in the file section file group with `@USE` attribute value "Representations" is described in the structural map as a single sub division.